tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -33,14 +35,15 @@ def should_exclude_files_type(granule_url: str, exclude_file_types: List[str]) -
 
 
 def copy_granule_between_buckets(source_bucket_name: str, source_key: str, destination_bucket: str,
-                                 destination_key: str) -> None:
+                                 destination_key: str, multipart_chunksize_mb: int) -> None:
```

Comment:
should `multipart_chunksize_mb` be of type `int` or `float`?

Reply:
Float. Fixed.

Reply:
int now